microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -18,17 +18,18 @@ Generally, extensions are distributed through the Microsoft Edge Add-ons website
 Extensions that aren't loaded from the Edge Add-ons store are referred to as externally installed extensions. The following are alternate methods of distributing externally installed extensions:
 
 *  Use the Windows registry (Windows only).
-*  Use a preferences JSON file (macOS and Linux).
+*  Use a preferences JSON file (macOS and Linux). The preferences JSON file would be your **Microsoft Edge extension's CRX ID**.json. The CRX ID is a unique 32 character code which consists of alphabets present at the end of your Microsoft Edge extension's URL. 
```

Comment:
"would" maybe should be "is."

Reply:
PR UI > Files Changed tab isn't showing the present comment.    Its link is 
https://github.com/MicrosoftDocs/edge-developer/pull/2272#discussion_r1007302740

Reply:
Needs engineering review

Reply:
The wording comes from this commit in the superseded PR 2101:
https://github.com/MicrosoftDocs/edge-developer/pull/2101/commits/f4b3d83e551211468959a2fd3f873b0d8aa931c3